src/sentry/templates/sentry/emails/feedback.txt

```diff
@@ -0,0 +1,48 @@
+{% spaceless %}
+{% autoescape off %}
+{% if enhanced_privacy %}
+Details about this feedback are not shown in this notification since enhanced
+privacy controls are enabled. For more details about this issue, view this
+issue on Sentry.
```

Comment:
```suggestion
privacy controls are enabled. For more details about this feedback, view this
feedback on Sentry.
```
could follow up with quick wording changes once emails are unbroken

Reply:
these are just used for debugging, so will follow up and make these edits.